app/controllers/mentor_profiles_controller.rb

```diff
@@ -16,7 +16,7 @@ def show
   def missing_mentee
     @user = current_user
     @mentee = current_user.partner
-    if @mentee.update_attribute(:is_missing, true)
+    if @mentee.update_attributes(is_missing: true, missing_since: DateTime.now)
```

Comment:
I have added now when mentee "report mentor missing", mentor is missing shown on both pages (mentee_show page, and mentor show page)